<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClipClass</class>
 <widget class="QDialog" name="ClipClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clip</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>输入图层：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_Input"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit_output">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_ok_">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>裁剪框类型</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_Type"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_output">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>84</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>输出文件位置：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_output">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>将结果输出为文件</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
